src/fetch.ts

```diff
@@ -194,7 +194,9 @@ export function createFetch(globalOptions: CreateFetchOptions = {}): $Fetch {
     }
 
     const hasBody =
-      context.response.body &&
+      (context.response.body ||
+        // https://github.com/JakeChampion/fetch/issues/1454
+        (context.response as any)._bodyInit) &&
```

Comment:
We need to handle this line now since there is a second source:

https://github.com/joshmossas/ofetch/blob/e11a6a0ca98196c239bd3f65c92be847abdc6898/src/fetch.ts

Reply:
@pi0 Okay I can make a quick patch

Reply:
Although I should mention, that streams still don't work properly on react native so idk if this is actually necessary.

Their fetch implementation is kinda jank lol

Reply:
Sure I can check real quick

Reply:
Thanks! Merging in between feel free to amend method if had any new findings.